erpnext/manufacturing/doctype/routing/routing.py

```diff
@@ -11,6 +11,18 @@
 class Routing(Document):
 	def validate(self):
 		self.set_routing_id()
+	
+	def on_update(self):
+		self.update_operation_time_in_linked_boms()
+
+	def update_operation_time_in_linked_boms(self):
+		bom_list = frappe.get_list("BOM", filters={"routing": self.name}, pluck="name")
+		for bom_no in bom_list:
+			for operation in self.operations:
+				frappe.db.sql("""
+					update `tabBOM Operation` set time_in_mins = %s
+					where operation = %s and parent = %s """,
```

Comment:
This updates the row-level time_in_mins but dependent values such as row-level operations cost and total operations cost are not updated.